source/sync/get-started.txt

```diff
@@ -125,8 +128,8 @@ determine whether users can read or write the objects that match the query.
 Define Data Access Rules
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Data access rules determine which data to sync, as well as each user's 
-ability to read and write data. Rules are closely linked to your app's data 
+Data access rules determine which data to sync, as well as each user's
+ability to read and write data. Rules are closely linked to your app's data
 model.
 
 With Flexible Sync, you specify which data to sync through queries for matching
```

Comment:
Just a nitpick - I think we can remove "Flexible" from "Sync" on these two pages, since it's no longer possible to make PBS apps (generally, except in weird edge cases) and we're not exposing the Sync Mode toggle anymore.

Reply:
Good catch!